tests/e2e/framework/kubernetes.go

```diff
@@ -97,21 +100,19 @@ func newKubeInfo(tmpDir, runID string) *KubeInfo {
 		Verbosity:  verbosity,
 		TmpDir:     tmpDir,
 		yamlDir:    filepath.Join(tmpDir, "yaml"),
-		Istioctl:   NewIstioctl(tmpDir, *namespace, *managerHub, *managerTag),
+		Istioctl:   istioctl,
```

Comment:
Those are not being used by kubernetes we can just add them in the main TestConfig. BUt let s do that in another PR because I need to merge to master.